docs/dev/automating-dco-checks.md

```diff
@@ -0,0 +1,22 @@
+# Automating DCO Checks
+
+Checking that each commit in a merge-request has a DCO sign-off
+can be automated. This should be done on the main upstream
+project for a project, not on forks. That means someone with appropriate
+permissions (e.g., a trustee or possibly a shop manager) must enable DCO checks.
+
+To enable DCO checks on GitHub, enable
+[Probot's DCO bot](https://probot.github.io/apps/dco/).
+Then, under `Settings` and `Branches`, you must protect `master` and turn on
+`Require status checks to pass before merging` and then select `DCO`.
```

Comment:
I tried doing this in a new test repository, is there something else you need to do to get DCO to show up as a status check besides installing the DCO bot and enabling it for the repository?

Reply:
Did you enable "status checks to pass before merging" and select "DCO" under that same section?

You know... I think a screen shot would be good... let me do that before you merge.

Reply:
I installed and enabled the DCO bot for all repositories. For some reason it's no showing up in that box.
![GitHub DCO status check](https://user-images.githubusercontent.com/47544822/62429220-d6d5b900-b6d9-11e9-87a4-15059ce55241.PNG)


Is there something else you did when setting up the DCO bot to enable it as a status check for the repository so it shows up here?

Reply:
Strange. So I installed it on an organization, LibreFoodPantry. I can go to LibreFoodPantry's `Settings` and `Installed GitHub Apps` and I see the following:

![Screen Shot 2019-08-05 at 8 40 57 AM](https://user-images.githubusercontent.com/1394670/62465404-ce729200-b75c-11e9-90ad-51a574e79826.png)

On this same page, I enabled DCO bot on specific repositories in LibreFoodPantry.

![Screen Shot 2019-08-05 at 8 41 54 AM](https://user-images.githubusercontent.com/1394670/62465512-0da0e300-b75d-11e9-930e-a43d03850119.png)

When I view one of the repositories' `Settings` and `Integrations & Services`, I see

![Screen Shot 2019-08-05 at 8 53 43 AM](https://user-images.githubusercontent.com/1394670/62466174-ac7a0f00-b75e-11e9-988d-18e951953ef6.png)

Does any of this help?

Reply:
I figured out the issue. In order for the DCO bot to show up as a status check in this window (under the `Branch protection rule` menu): 
![image](https://user-images.githubusercontent.com/47544822/62498063-f50be980-b7ab-11e9-8567-da25c31904ce.png)

You have to have triggered the bot at least once previously (in the past week). You can do this by creating a new branch and a commit then making a pull request to the master branch where the DCO bot will then run for the first time. After this, it will then show up as a status check under the branch protection settings where you can then enable it. 

This issue doesn't appear to be exclusive to the DCO bot. It seems that you have to have whatever GitHub app you want to run as a status to check to run at least one time before it will show up in the status checks box where you can then enable it as a branch protection as suggested by this CI article: 
https://medium.com/meshstudio/continuous-integration-with-circleci-and-nodejs-44c3cf0074a0